src/components/HomePage.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import MovieCard from "./MovieCard";
+
+import { Container } from "../styles/Container.styled";
+import GlobalStyles from "../styles/Global.styled";
+import { Cards } from "../styles/Cards.styled"
+import PageNotFound from "./PageNotFound";
+import {apiUrl,searchUrl} from "./constants/Global"
+
+
+function HomePage(){
+  const [movies, setMovies] = useState([]);
+  const [searchTerm, setSearchTerm] = useState("");
+
+  useEffect(() => {
+    getMovies(apiUrl)
+  },
+    [])
+
+  async function getMovies(url) {
+    const response = await axios.get(url);
+    const result = response.data.results;
+
+    setMovies(result);
+  }
+
+  const submit = (e) => {
```

Comment:
Please be consistent with syntax. either choose function keyword and use it across the app or choose arrow function syntax and use it across the app. Do not mix both.